litefs/disaster-recovery.html.md.erb

```diff
@@ -0,0 +1,174 @@
+</div>
+
+## Start your app up with only the primary node
+
+It's simpler to recover a single node, rather than several nodes, so to
+start out with, it's best to remove all replica nodes and run just a
+primary node. If your app has already restarted, you may need to temporarily
+remove some replicas. If you're running your app on the Fly Platform, you can
+see the machines running your app with:
+
+```sh
+fly status
+```
+
+You'll see something like this:
+
+```
+Machines
+PROCESS	ID            	VERSION	REGION	STATE  	ROLE   	CHECKS	LAST UPDATED
+app    	784e900c2de378	1      	ams   	started	primary	      	2023-07-10T13:05:40Z
+app    	e784e471b39208	1      	ord   	started	replica	      	2023-07-10T13:01:16Z
+```
+
+Then destroy each machine that's marked `replica`:
+
+```sh
+fly machines destroy e784e471b39208
+```
+
+You can re-run `fly status` after you're done, to confirm you only have the primary running.
+
+## Find the correct cluster id
+
+When your app restarted, LiteFS generated a new cluster id, which is different
+than the cluster id stored in LiteFS Cloud. This is a safety measure: we don't want
+to accidentally overwrite data in a cluster if you inadvertently set the wrong LiteFS
+Cloud token for the app. You'll see some error messages in the LiteFS logs, similar
+to this:
+
+```
+level=INFO msg="backup stream failed, retrying: fetch position map: backup client error (422): cluster id mismatch, expecting \"LFSC25AD000F32ED02A9\""
+```
+
+If you're running on the Fly platform, you can see the logs with:
+
+```sh
+fly logs
+```
+
+Note the cluster id specified at the end of the log message: `LFSC25AD000F32ED02A9` in
+our example. Copy this value to use later.
+
+## Update the cluster id
+
+Now that you have the correct cluster id, you can connect to your primary node and
+update the cluster id file. If your app is deployed on the Fly Platform, you can
+connect with:
+
+```sh
+fly ssh console
+```
+
+Then update the `clusterid` file:
+
+```sh
+echo LFSC25AD000F32ED02A9 > /var/lib/litefs/clusterid
+```
+
+After this, you should restart litefs. If you're using the Fly Platform, close
+your ssh session, and then restart your app:
+
+```sh
+fly app restart
+```
+
+Check your logs again, and confirm you don't see the error message anymore!
+You should also take a look at your app and confirm you see the data you expect
+before you continue.
+
+## Resolve consul key error
+
+If you're running on the Fly Platform (or you're using Consul for lease management
+elsewhere), you may see the following errors in your logs:
+
+```
+2023-07-10T13:30:25Z app[784e900c2de378] ams [info]level=INFO msg="cannot connect, \"consul\" lease already initialized with different ID: LFSCA5E3680C1C2FCFFE"
+```
+
+You have two options to resolve this:
+
+- change the Consul key value that LiteFS uses. This is easy, but somewhat hacky/ugly.
+- remove the "wrong" key from Consul. This is more difficult but cleaner.
```

Comment:
I worry about using the phrase "hacky/ugly". It makes it sound like a bad option but the only real tradeoff is that you have extra Consul keys. Can we rephrase this to simply state the tradeoff rather than say one is better than another?